src/test/java/org/ecommerce/repositories/jpa/OrderRepositoryJpaImplTest.java

```diff
@@ -16,16 +14,17 @@
 import java.util.Comparator;
 import java.util.Date;
 import java.util.List;
+import java.util.NoSuchElementException;
 
 import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.Mockito.lenient;
 
 @ActiveProfiles("local")
 @SpringBootTest
-class OrderServiceJpaImplTest {
+class OrderRepositoryJpaImplTest {
```

Comment:
Please rename to `OrderJpaRepositoryTest`

Reply:
Done https://github.com/algutierrez-griddynamics/e-commerce/pull/51/commits/d21d67c0ff631e4bbb78eca9c1e8b0e5d731589b